src/nodes/hypertable_modify.c

```diff
@@ -524,8 +571,48 @@ static CustomPathMethods hypertable_modify_path_methods = {
 	.PlanCustomPath = hypertable_modify_plan_create,
 };
 
+#if PG14_GE
+static bool
+should_chunk_append(Hypertable *ht, PlannerInfo *root, RelOptInfo *rel, Path *path)
```

Comment:
This name was somewhat confusing when I first read the code, I'd call it 'use_chunk_append_path'